### PR TITLE
Bulk Geocode - Properly pass thorugh `params`

### PR DESCRIPTION
**Pull request by gavinr**
#630

### CODE REVIEW

**Comment by gavinr** [comment]
@jgravois would love your feedback on this code change if you have time.

File: packages/arcgis-rest-geocoding/src/bulk.ts [hunk 1]
```diff
@@ -66,13 +66,18 @@ export function bulkGeocode(
...
-      forStorage: true,
-      addresses: { records: [] }
-    },
+    params: {},
     ...requestOptions
   };
 
+  if (!options.params.forStorage) {
+    options.params.forStorage = true;
+  }
```
Comment (jgravois) [inline packages/arcgis-rest-geocoding/src/bulk.ts]: I know its against the TOU and I _think_ that failing to set `forStorage: true` will cause the call to fail. I'd recommend removing the if statement entirely.

other than that, this should be good to go. 😎

Comment (gavinr) [reply packages/arcgis-rest-geocoding/src/bulk.ts]: @jgravois after looking into this, I found the documentation that supports your point:

> It is assumed that results from this operation are always stored for later use, so forStorage is an invalid parameter and shouldn't be added to a request.

image
https://developers.arcgis.com/rest/geocode/api-reference/geocoding-free-vs-paid.htm#ESRI_SECTION1_B9D659BA26584F26A65CBA24E59AFE6E

Thanks for the feedback!

**Comment by jgravois** [comment]
I'm glad you didn't tell him to get clever and do:
```ts
options.params.addresses = {
  records: options.addresses.map(attributes => { attributes })
};
```
#forEach()4lyfe

**Comment by tomwayson** [comment]
Actually @gavinr do you mind using `map` as @jgravois suggests above and then doing `npm run c` to generate a commit message that will make it easier for me to generate the changelog? then I will merge and deploy.

**Pull request merged**